### PR TITLE
Resend transaction #115

### PR DESCRIPTION
**Pull request by epexa**
#115

### CODE REVIEW

File: src/redux/wallet.js [hunk 1]
```diff
@@ -759,5 +761,52 @@ export const resend = (txHash) => (dispatch, getState) => {
...
+        console.log('RESEND', transaction);
+        dispatch(showSigner({
+            fromAddress: transaction.from,
+            toAddress: transaction.to,
+            gasPrice: 20,
+            gasLimit: 210000,
+            value: transaction.amount,
+        })).then((data) => {
+            console.log('>>', data, web3.utils.toWei(`${data.gasPrice}`, 'Gwei'));
+            eth.sendTransaction({
```
Comment (Pechalka) [inline src/redux/wallet.js:847]: where data and nonce? it will not work

Comment (epexa) [reply src/redux/wallet.js:847]: fixed nonce and data

**Comment by mastercyb** [comment]
@epexa Please name your pull requests

**Comment by epexa** [comment]
> @epexa Please name your pull requests

@xhipster  done

**Pull request merged**